lib/matplotlib/cbook.py

```diff
@@ -1941,8 +1941,8 @@ def _bootstrap_median(data, N=5000):
         M = len(data)
         percentiles = [2.5, 97.5]
 
-        ii = np.random.randint(M, size=(N, M))
-        bsData = x[ii]
+        bs_index = np.random.randint(M, size=(N, M))
+        bsData = data[bs_index]
```

Comment:
np.random.choice(<array>, ...) is basically implemented as array[np.random.randint(...)] so it's the same.